plugins/terms/plugin_tests/termsSpec.js

```diff
@@ -79,10 +89,10 @@ describe('Navigate to a non-collection folder and item', function () {
 
     it('create an item', function () {
         runs(function () {
-            return $('.g-create-item').click();
+            $('.g-create-item').click();
         });
-        // This causes the test to fail
-        // girderTest.waitForDialog();
+        girderTest.waitForDialog();
```

Comment:
@manthey To be clear, the reason you believe this line was failing before was that the page state was typically synchronized from the test by this point?

Reply:
Before, we failed to wait for the page transition to `#folder/id` before invoking the dialog.  If we waited for the dialog, that transition eventually could trigger and remove the dialog, causing this to fail.  Now, we properly wait for `#folder/id` and then open a dialog so we can properly wait for the dialog.